magento/pwa-studio
# PWA-706: Update Venia's cards to match the style guides cards

## Description

Adjust `Card` styles to match the new design (mainly used on Cart & checkout)

## Related Issue





https://jira.corp.magento.com/browse/PWA-706

## Acceptance 



### Verification Stakeholders

@jimbo
@dpatil-magento
@anthoula
### Specification



### Verification Steps


1. Navigate to checkout, checkout address book and verify that cards meet the style guide
2. Verify all card instances are working as expected

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
